stack/base/load-singlesshagent.sh

```diff
@@ -32,7 +32,7 @@ load_singlesshagent() {
         [ "$VERBOSE" == "true" ] && echo "- no existing environment to source" >&2
     fi
 
-    SSH_ADD_OUTPUT=`ssh-add -l`
+    SSH_ADD_OUTPUT=`ssh-add -l 2> /dev/null`
```

Comment:
@unkcpz here's the small change. Note that it is okay to redirect error to dev/null here since we check the return value of this `ssh-add` command below.